doc/format.md

```diff
@@ -145,13 +145,18 @@ It can be referenced in other rules, so if you change a rule name, be sure to se
 By convention, the rule name should complete one of the following sentences:
   - "The program/function may..."
   - "The program was..."
-  
+
+To focus rule names we try to omit articles (the/a/an).

```

Comment:
My preference, but happy to debate this.

Reply:
```suggestion
To focus rule names we try to omit articles (the/a/an).
For example, prefer `make HTTP request` over `make an HTTP request`.
```